.travis.yml

```diff
@@ -26,8 +26,9 @@ install:
 - npm install
 before_script:
 # Install Firefox Nightly to prevent signing errors.
+- npm i -g get-firefox
```

Comment:
It can yes. Would you prefer to then call the binary from the `.travis.yml` or with an npm script?